docs/docsite/rst/playbook_guide/playbooks_filters.rst

```diff
@@ -2021,6 +2021,18 @@ To join one or more path components:
 
 .. versionadded:: 2.10
 
+.. code-block:: yaml+jinja
+
+    {{ path | commonpath }}
+
+To get Normalize path, eliminating double slashes, etc. (new in version 2.10):
```

Comment:
1) The next release is 2.14 and that's when it might be added
2) `.. versionadded::` Sphinx directive must be used for such information, consistent with other places.

Reply:
don't bother with this page, i'm removing most content soon, see this as the 'new way' to document filters/tests https://github.com/ansible/ansible/blob/devel/lib/ansible/plugins/filter/dirname.yml

Reply:
So do I need to make the changes or not?

Reply:
not the ones on playbooks_filters page, but you need to create the .yml documentation

Reply:
Understood will make the change